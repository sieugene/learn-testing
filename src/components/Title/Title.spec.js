import React from 'react'
import { shallow } from 'enzyme';
import Title from './Title';

describe("Title component Render", () => {
    it("should render component with Props", () => {
        const component = shallow(<Title title={'test title'}/>)
        const wrap = component.find('.title')
        expect(wrap.text()).toBe('test title')
        expect(component).toMatchSnapshot()
    })
    it("should render component without Props", () => {
        const component = shallow(<Title/>)
        expect(component).toMatchSnapshot()
    })
})