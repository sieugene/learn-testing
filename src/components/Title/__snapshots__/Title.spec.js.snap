// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component Render should render component with Props 1`] = `
<h1
  className="title"
>
  test title
</h1>
`;

exports[`Title component Render should render component without Props 1`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
